en/configure-a-tidb-cluster.md

```diff
@@ -127,6 +127,22 @@ If you want to enable TiCDC in the cluster, you can add TiCDC spec to the `TiDBC
       replicas: 3
 ```
 
+#### Deploy the enterprise version
```

Comment:
Looks like this part is missing:
![image](https://user-images.githubusercontent.com/41534398/88755227-3c487280-d193-11ea-8012-66eb50b95ea6.png)

Reply:
It's moved to L121~L128. PTAL